main.go

```diff
@@ -153,6 +153,9 @@ func main() {
 	}
 	log.Printf("PPL: -- %+v -- ", ppl)
 
+	// Provide a date string for three days in the future
+	threeDays := time.Now().AddDate(0, 0, 3).Format(time.RFC850)
```

Comment:
Does the Secureframe API return some sort of expiration date or time that the test started failing? I'd rather rebase on that, because otherwise the deadline will forever move forward 72-hours.

Reply:
Nope :(

Here's an example (scrubbed) response for a user with expired/incomplete training:
```
{
      "id": "...",
      "type": "user",
      "attributes": {
        "created_at": "2024-09-24T01:22:15.686Z",
        "updated_at": "2024-10-01T00:01:28.448Z",
        "id": "...",
        "active": true,
        "active_source": "...",
        "department_id": null,
        "employee_type": "...",
        "end_date": null,
        "in_audit_scope": true,
        "invited": true,
        "invited_at": "2024-09-30T16:00:54.708Z",
        "onboarding_status": "security_training",
        "personnel_status": "incomplete_tasks",
        "role": null,
        "secureframe_agent_acknowledged_at": null,
        "start_date": null,
        "title": null,
        "access_role": null,
        "email": "user@domain.com",
        "first_name": "First",
        "image_url": null,
        "last_name": "Last",
        "manager_name": null,
        "middle_name": "",
        "name": "First Last",
        "preferred_first_name": null
      },
      "relationships": {},
      "links": {
        "self": "https://api.secureframe.com/users/..."
      }
},
```

The only date fields are:
- `created_at`
- `updated_at`
- `invited_at`

As far as `personnel_status` is concerned, we can't see the individual incomplete tasks via the REST API either.

A `GET` on that specific user ID contains no additional information, either.